config/settings.yml

```diff
@@ -1211,6 +1211,7 @@
       :vim_performance_states_purge_interval: 1.day
       :vm_retired_interval: 10.minutes
       :yum_update_check: 12.hours
+      :audit_managed_resources: 1.days
```

Comment:
this list is intended to be alphabetized, so if you can move this line into the proper place - that would be great

Reply:
I moved my setting to the top of the list. Although the list isn't sorted properly. Should I move the other settings?

Reply:
let's change the other out of order ones in a separate PR

Reply:
I didn't find a way other then restarting `evmserverd` for this value (when changed) to pick up. Is this expected?